pcsx2/ImGui/FullscreenUI.cpp

```diff
@@ -1677,6 +1677,7 @@ void FullscreenUI::SetSettingsChanged(SettingsInterface* bsi)
 		s_game_settings_changed.store(true, std::memory_order_release);
 	else
 		s_settings_changed.store(true, std::memory_order_release);
+	Host::OnSettingsChangedExternally();
```

Comment:
Can you separate these with a line, it's bunched up.

Reply:
Done